install_dependencies.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+sudo apt install git curl clang-12 -y
+
+mkdir dependencies
+
+git clone git@github.com:rcohencyberarmor/kubescape-ebpf-engine.git dependencies/kubescape_ebpf_engine_sc
```

Comment:
[Required]: Move this repo to Kubescape org. It is required a PR and PR approval before we use it in production